docs/basics/quick_start_rl/quickstart.md

```diff
@@ -221,25 +220,21 @@ cannot compute the optimal policy, we could simply compare the rewards
 gathered during learning, instead of the regret.
 
 First, we have to record the reward during the fit as this is not done
-automatically. To do this, we can use the
-[WriterWrapper](rlberry.wrappers.writer_utils.WriterWrapper)
-module, or simply the [writer](rlberry.agents.Agent.writer) attribute.
+automatically. To do this, we can use the `writer_extra` optional parameter.
 
 ```python
 class RandomAgent2(RandomAgent):
     name = "RandomAgent2"
 
     def __init__(self, env, **kwargs):
-        RandomAgent.__init__(self, env, **kwargs)
-        self.env = WriterWrapper(self.env, self.writer, write_scalar="reward")
+        RandomAgent.__init__(self, env, writer_extra="reward", **kwargs)
 
 
 class UCBVIAgent2(UCBVIAgent):
     name = "UCBVIAgent2"
 
     def __init__(self, env, **kwargs):
-        UCBVIAgent.__init__(self, env, **kwargs)
-        self.env = WriterWrapper(self.env, self.writer, write_scalar="reward")
+        UCBVIAgent.__init__(self, env, writer_extra="reward", **kwargs)
```

Comment:
This can be simpler. For instance, for UCBVIAgent, it is only using the ExperimentManager with init_kwargs containing "reward":
```
ucbvi_params["writer_extra"] = "reward"
ucbvi_stats = ExperimentManager(
    UCBVIAgent2,
    (env_ctor, env_kwargs),
    fit_budget=50,
    init_kwargs=,
    n_fit=10,
)
ucbvi_stats.fit()
```

Reply:
done